blocks/v2-pencil-promo/v2-pencil-promo.js

```diff
@@ -0,0 +1,86 @@
+import {
+  getImageURLs,
+  createResponsivePicture,
+  variantsClassesToBEM,
+} from '../../scripts/common.js';
+
+export default async function decorate(block) {
+  const blockName = 'v2-pencil-promo';
+  const variantClasses = ['pencil-banner-black', 'pencil-banner-copper', 'promo-banner-gold', 'promo-banner-copper'];
+  variantsClassesToBEM(block.classList, variantClasses, blockName);
+
+  const pencilBanners = document.querySelectorAll('.v2-pencil-promo--pencil-banner-black, .v2-pencil-promo--pencil-banner-copper');
```

Comment:
you can use `${blockName}--pencil-banner-black` in this and next line

Reply:
I solved that using another approach that includes pencil or promo variants